### ISSUE TITLE
i'm not able to install the package

### ISSUE THREAD

**Issue by chaosLegacy** [raynor85/react-native-material-selectize#13]
Hi, I want to use your library, but I can't download it via npm either yarn
- For yarn its stuck on Fetching packages...

<img width="1041" alt="screen shot 2018-09-09 at 1 39 33 pm" src="https://user-images.githubusercontent.com/11313734/45264561-7f31bd80-b436-11e8-945b-5c0532a87a9e.png">

- For npm its stuck on react-native-material-selectize@1.11.0

<img width="668" alt="screen shot 2018-09-09 at 1 41 13 pm" src="https://user-images.githubusercontent.com/11313734/45264562-7f31bd80-b436-11e8-8920-cea672a8a0ec.png">

 I also tried installing pervious versions of the package like 1.10.0 and 1.9.0 but no luck.
PS: It's not an internet problem, my internet is working fine.

Please help.

**Comment by raynor85** [comment]
I just tried to install it using yard add react-native-material-selectize, and it worked fine. You can always add the github repo directly inside the package.json if npm.js it troubling you

**Comment by chaosLegacy** [comment]
Yeah, now it working it took about 13 min to finish downloading the package,
little bit weird
Everything is okay now Thanks a lot.

**Comment by chaosLegacy** [comment]
Is there a way to close the dropdown list on blur (tap outside)?
it seems that the list stay visible till the user tap on another input 
In this example the dropdown list refuse to hide till i tap on another input 
<img width="287" alt="screen shot 2018-09-09 at 4 33 40 pm" src="https://user-images.githubusercontent.com/11313734/45266139-279f4c00-b44e-11e8-8e88-6de4eda7353e.png">

an other little issue related to the UI is that as soon as i keep adding new items the screen padding start shrinking and the content overlaps see

<img width="288" alt="screen shot 2018-09-09 at 4 36 11 pm" src="https://user-images.githubusercontent.com/11313734/45266166-8a90e300-b44e-11e8-834a-0f78beefd812.png">


Thank you.
